Add feature show/hide description

PhMemmel opened the pull request:
Closes #69

Syxton commented:
@PhMemmel This looks really good.  The only slight issue I saw was with the behat tests.  Unlike your unit tests which check to see if the descriptions are hidden at first, then shows them and checks, then hides them and checks, the behat tests assume they are hidden from the start but doesn't check that the descriptions are hidden before doing the show descriptions action.

PhMemmel commented:
Thanks @Syxton for the review.
I rebased the branch, added the default value check you suggested, ran behat locally against all currently supported formats.
